datastore/database.go

```diff
@@ -53,17 +53,16 @@ type Datastore interface {
 	CreateSigningLogTable() error
 	CheckForDuplicate(signLog *SigningLog) (bool, int, error)
 	CreateSigningLog(signLog SigningLog) error
-	ListSigningLog() ([]SigningLog, error)
-	DeleteSigningLog(signingLog SigningLog) (string, error)
```

Comment:
This was initially included as it was not allowed to get a serial assertion for the same device more than once. This allowed developers to delete the log entry and reflash the device. Now you can actually get a serial assertion for the same device multiple times and you get a bumped revision number. So there is no longer a need to delete a signing log entry.

The feature was removed from the UI a long time ago... this is cleaning up the backend.